### DISCUSSION TITLE
More volume control areas

### DISCUSSION THREAD

**Discussion by SheepCommander** [ppy/osu#26190] [Ideas] [answered]
#### Currently, scrolling the scrollwheel on the areas below does not do anything
![image](https://github.com/ppy/osu/assets/66646514/19204fb5-8c82-4c5a-8b53-c752a323f2e1)

### I think it would be a great idea to **allow those areas to activate volume controls**, just like in the main menu
![image](https://github.com/ppy/osu/assets/66646514/b8f3cabe-f188-4276-9c0e-41292bb515aa)

---
_while searching this issue, I learned that there alt+scroll is also an option, but I think this would be much more intuitive and useful for users who navigate with their mouse._

**Comment by peppy** [comment, marked as answer]
This will not happen. You can hover the music icon in the toolbar to change volume, or you can bind mousewheel + control or something to it in settings.
